dlt/destinations/impl/databricks/configuration.py

```diff
@@ -37,9 +36,32 @@ class DatabricksCredentials(CredentialsConfiguration):
 
     def on_resolved(self) -> None:
         if not ((self.client_id and self.client_secret) or self.access_token):
+            try:
+                # attempt notebook context authentication
+                from databricks.sdk import WorkspaceClient
+                from databricks.sdk.service.sql import EndpointInfo
+
+                w = WorkspaceClient()
+                self.access_token = w.dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().getOrElse(None)  # type: ignore[union-attr]
+
+                # pick the first warehouse on the list
```

Comment:
please take a list of warehouses when `server_hostname` or `http_path` and then set only the empty. configuration takes precedence. and if we have both values we skip getting warehouses altogether

Reply:
done